Formula/o/ossp-uuid.rb

```diff
@@ -1,14 +1,16 @@
 class OsspUuid < Formula
   desc "ISO-C API and CLI for generating UUIDs"
-  homepage "http://www.ossp.org/pkg/lib/uuid/"
-  url "https://deb.debian.org/debian/pool/main/o/ossp-uuid/ossp-uuid_1.6.2.orig.tar.gz"
-  sha256 "11a615225baa5f8bb686824423f50e4427acd3f70d394765bdff32801f0fd5b0"
+  homepage "https://git.sr.ht/~nabijaczleweli/ossp-uuid"
+  url "https://git.sr.ht/~nabijaczleweli/ossp-uuid/archive/UUID_1_6_3.tar.gz"
+  sha256 "630b7480b63c5c3e298fe1e3c39469440cd5fca4e13ba04690fba401fdab64f8"
```

Comment:
Only Debian (and a few other small distros) so far has picked this up: https://repology.org/project/ossp-uuid/versions

Do we want to wait and see what others do first?

Reply:
that works for me 👍

Reply:
Maybe I'm being overly careful here, but from these sources:

- `https://github.com/gentoo/gentoo/pull/38536`
- https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1081555,
- https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=281427

It seems, especially from the FreeBSD ticket above, that this new upstream has not been officially endorsed by the original OSSP authors, and most if not all repository location updates in other distros are submitted by the new author.